config/manifests/bases/backstage-operator.clusterserviceversion.yaml

```diff
@@ -17,7 +17,10 @@ spec:
       kind: Backstage
       name: backstages.rhdh.redhat.com
       version: v1alpha1
-  description: Operator to deploy Backstage on Kubernetes
+  description: |
+    Operator to deploy Backstage on Kubernetes
+
+    The telemetry data collection feature is enabled by default. Red Hat Developer Hub sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh
```

Comment:
Same suggestion as above:

```suggestion
    The telemetry data collection feature is enabled by default. In the default configuration of the operator, telemetry data will be sent to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#telemetry-collection
```

> I would prefer to move operator part from showcase document to operator repo (as TELEMETRY.md or so) and maintain it here. The reason is that we are evolving and I'd like to make sure the info is updated.

Even if this [doc](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#disable-telemetry) mentions the operator, I read it as more about disabling telemetry in the Showcase app, not telemetry from the operator itself.
So I think it makes sense to be in the Showcase docs, as the operator may need to support multiple versions of the Showcase app (and the way telemetry is configured in the app might be different between 2 versions of the app).
Maybe we will need to also send telemetry data from the operator itself later, in which case a `TELEMETRY.md` file here would make sense, I guess.

WDYT?

Reply:
Yes, it is Showcase's configuration but user configures it using Operator (in this case)
The way Operator makes this configuration is Operator specific and may be changed or new options may be introduced in the future.
So, IMO, it will be simpler to maintain it on Operator side.

Reply:
> Yes, it is Showcase's configuration but user configures it using Operator (in this case) The way Operator makes this configuration is Operator specific and may be changed or new options may be introduced in the future. So, IMO, it will be simpler to maintain it on Operator side.
> 
> In other words: if we keep Operator specific configuration in Showcase repo, who will maintain it? :)

I understand your point, but this can indeed evolve in 2 ways:
- either the Showcase changes the way telemetry is configured. In this case, I think it makes sense to have the related doc in the Showcase repo.
- or the Operator changes the way dynamic plugins are defined in the Custom Resource (which can have an impact on how telemetry is configured). In this case, I agree with you that it makes sense to have the doc in the Operator repo.

Right now, I see it as documenting the specific Showcase telemetry using all possible installation methods on the same doc. And the Operator has a more general section about configuring dynamic plugins: https://github.com/janus-idp/operator/blob/main/.rhdh/docs/openshift.adoc#configuring-dynamic-plugins-with-the-operator

But your question is more general to me and should be handled outside of this PR.
As this doc was added in https://github.com/janus-idp/backstage-showcase/pull/1028 (https://issues.redhat.com/browse/RHIDP-1311), let's file a separate issue for this if you think the doc content should be reorganized.

Reply:
My question is not that general :), it is as simple as: who and how will maintain actuality of Operator configuration for this case?
As for your examples:
- if something changed on Showcase, Operator will have to react and change something too (including doc apparently)
- but not opposite, i.e. if something changed on Operator, most probably Showcase need not to react.
That's why Showcase part of document should rather contain a reference to Operator/Helm specific and not Operator/Helm specific information itself.
Does it make sense?

Reply:
> My question is not that general :), it is as simple as: who and how will maintain actuality of Operator configuration for this case?
>
In this case, the person who made the change will be responsible for ensuring the Showcase docs is also updated.
But I said "general" because I saw this pattern in almost all Showcase docs (like the docs for adding monitoring, logging, corporate proxy - see https://github.com/janus-idp/backstage-showcase/tree/main/showcase-docs).

If tomorrow we decide to make telemetry opt-in by default in the Showcase app, your suggestion means we would have to create 3 different PRs (one in Showcase, and one per installation method), right? Rather than a single PR in the Showcase repo?